tests/testthat/test-external-generic.R

```diff
@@ -0,0 +1,28 @@
+test_that("new_method works with both hard and soft dependencies", {
+  skip_on_os("windows")
+  skip_if(quick_test())
```

Comment:
In case you are annoyed about the tests taking a while, you can set `quick_test_enable()` to disable this particular test, which takes a while because it has to install a bunch of packages. It also sometimes fails until you actually install R7 depending on what you changed. I usually turn on the quick tests it when I am working on R7.